src/data/faq.json

```diff
@@ -51,10 +51,10 @@
                         "anchor": "error_log",
                         "active": true,
                         "textblock": [
-                            "With version 2.2, the Corona-Warn-App (CWA) offers a new function for detailed technical problem analysis. The error report is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the technical hotline (national: 0800 7540001 or international: +49 30 498 75401) to verify if there is already a known solution. An error report is only required if the CWA team asks for this report.",

```

Comment:
> The error report is intended to help the developers to **faster** perform a root cause analysis

better: The error report is intended to help the developers to perform a root cause analysis **faster**

Reply:
Done in https://github.com/corona-warn-app/cwa-website/pull/1267/commits/c9e9c06a81bb919a70e665c160750f76837d0a5d